tasks/main.yml

```diff
@@ -44,6 +52,14 @@
   environment: project_environment
   when: project_has_npm
 
+- name: Check if bower dir exists
+  stat: path={{ deploy_helper.current_path }}/{{ project_bower_components_path }}
+  register: check_bower_components_path
+
+- name: Copy bower dir if exists to speed up bower install
+  command: /bin/cp -rp {{ deploy_helper.current_path }}/{{ project_bower_components_path }} {{ deploy_helper.new_release_path }}/{{ project_bower_components_path }}
```

Comment:
Why is `project_bower_components_path` appended to the destination path, while this is not done for composer (on line 42)? Going either way is fine, but lets keep it consistent :smile: